src/liquidity.rs

```diff
@@ -130,7 +130,18 @@ pub struct LSPS2ServiceConfig {
 	pub min_payment_size_msat: u64,
 	/// The maximum payment size that we will accept when opening a channel.
 	pub max_payment_size_msat: u64,
-	/// Use the client trusts lsp model
+	/// Use the 'client-trusts-LSP' trust model.
+	///
+	/// When set, the service will delay *broadcasting* the JIT channel's funding transaction until
+	/// the client claimed sufficient HTLC parts to pay for the channel open.
+	///
+	/// Note this will render the flow incompatible with clients utilizing the 'LSP-trust-client'
```

Comment:
Perhaps make it more clear that this bool switches between the two models, and that it should be false for LSP-trust-client. Maybe an enum would have been better here?

Reply:
Why offer the boolean then if there is no demand yet and you think it shouldn't even be in the spec?